install/cluster-operator/060-Deployment-strimzi-cluster-operator.yaml

```diff
@@ -42,6 +42,8 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.namespace
+            - name: STRIMZI_OPERATIONS_THREAD_POOL_SIZE
+              value: "10"
```

Comment:
* The `install/` files are released installation files. You should not change them as a regular PRs. The once to be changed are in `packaging/` (+ the 060... file is actually generated from the Helm Chart)
* But I do not think we expect the users to change these. So I would not add the defaults to the deployments anyway. I would just remove it from here.